ydb/core/tx/schemeshard/schemeshard__operation_copy_sequence.cpp

```diff
@@ -274,6 +274,50 @@ class TProposedCopySequence : public TSubOperationState {
                 << " operationId#" << OperationId;
     }
 
+    void UpdateSequenceDescription(NKikimrSchemeOp::TSequenceDescription& descr) {
+        descr.SetStartValue(GetSequenceResult.GetNextValue());
```

Comment:
I think I said it already, that you can't store current values in the schema. Current sequence value is data, not schema. When you describe table before backing up, then backup and restore, you should see the same describe table result as you did before backing up.